bundles/framework/layerlist/resources/locale/en.js

```diff
@@ -37,7 +37,10 @@ Oskari.registerLocalization(
                 'inspire': 'By Theme',
                 'organization': 'By Data Provider',
                 'noGroup' : 'No group',
-                'manyLayersWarn': 'Warning, you are adding 10 or more layers to the map. Continue?'
+                'manyLayersWarn': 'Warning, you are adding 10 or more layers to the map. Continue?',
+                'providers': {
+                    'userlayer': 'Own datasets'
+                }
```

Comment:
We can revert this, since the functionalities themselves create the localized name for data provider